configure

```diff
@@ -67,6 +67,13 @@ save_configuration
 
 freshen_chef_repo
 
-cd chef_repo
+pushd chef_repo
 wd="$(pwd)"
-exec cinc-solo -c "${wd}/.chef/solo.rb" -E "$CHEF_ENVIRONMENT" -j "${wd}/${CHEF_SOLO_FILE}"
+cinc-solo -c "${wd}/.chef/solo.rb" -E "$CHEF_ENVIRONMENT" -j "${wd}/${CHEF_SOLO_FILE}"
+
+# Write date + commit SHA to a changelog
+echo "$( date "+%Y-%m-%d %H:%M:%S"):  $(git rev-parse --short HEAD)" >> "../CHEF_CHANGELOG.txt"; 
+popd 
+
+# Remove chef repository after configuration
+sudo rm -r chef_repo
```

Comment:
Why is this done with sudo?

I have specifically avoided use of sudo within these scripts, instead preferring to elevate permissions before the script is run since I am personally not a fan of using sudo to elevate within scripts, preferring instead that the user should elevate explicitly and the script should de-elevate when running operations as a non-privileged user.